dialogue-client-verifier/src/test/java/com/palantir/verification/IntegrationTest.java

```diff
@@ -112,6 +121,62 @@ public void conjure_generated_blocking_interface_with_optional_binary_return_typ
         assertThat(maybeBinary.get()).hasSameContentAs(asInputStream("Hello, world"));
     }
 
+    @Test
+    public void can_clients_abort_in_progress_server_work() throws Exception {
```

Comment:
Are we thinking about propagating timeouts for calls to the server? With http1 will cancelling the future not cost us connections?